integration/tests/tracer/test_tracer_debug_trace_transactions_callTracer.py

```diff
@@ -0,0 +1,501 @@
+            lambda: self.tracer_api.send_rpc(
+                method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+            )["result"]
+            is not None,
+            timeout_sec=120,
+        )
+
+        response = self.tracer_api.send_rpc(
+            method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+        )
+
+        expected_response["from"] = receipt["from"].lower()
+        expected_response["to"] = receipt["contractAddress"].lower()
+        expected_response["gasUsed"] =  hex(receipt["gasUsed"])
+        expected_response["type"] = "CREATE"
+
+        self.assert_response_contains_expected(expected_response, response)
+
+    def test_callTracer_type_call(self, storage_contract):
+        sender_account = self.accounts[0]
+        store_value = random.randint(1, 100)
+        tracer_params = { "tracer": "callTracer", "tracerConfig": { "onlyTopCall": True } }
+        expected_response = {}
+
+        tx_obj, _, receipt = call_storage(sender_account, storage_contract, store_value, "blockNumber", self.web3_client)
+        
+        wait_condition(
+            lambda: self.tracer_api.send_rpc(
+                method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+            )["result"]
+            is not None,
+            timeout_sec=120,
+        )
+
+        response = self.tracer_api.send_rpc(
+            method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+        )
+
+        expected_response["from"] = tx_obj["from"].lower()
+        expected_response["to"] = tx_obj["to"].lower()
+        expected_response["gasUsed"] =  hex(receipt["gasUsed"])
+        expected_response["input"] = tx_obj["data"]
+        expected_response["type"] = "CALL"
+
+        self.assert_response_contains_expected(expected_response, response)
+    
+    def test_callTracer_withLog_check(self, event_caller_contract):
+        sender_account = self.accounts[0]
+        expected_response = {}
+
+        tx = self.web3_client.make_raw_tx(from_=sender_account)
+        instruction_tx = event_caller_contract.functions.callEvent1("Event call").build_transaction(tx)
+        receipt = self.web3_client.send_transaction(sender_account, instruction_tx)
+
+        tracer_params = { "tracer": "callTracer", "tracerConfig": { "withLog": True } }
+
+        wait_condition(
+            lambda: self.tracer_api.send_rpc(
+                method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+            )["result"]
+            is not None,
+            timeout_sec=120,
+        )
+
+        response = self.tracer_api.send_rpc(
+            method="debug_traceTransaction", params=[receipt["transactionHash"].hex(), tracer_params]
+        )
+
+        expected_response["from"] = instruction_tx["from"].lower()
+        expected_response["to"] = instruction_tx["to"].lower()
```

Comment:
Isn't fill_expected_response appropriate for some tests?

Reply:
I fixed this, I added fill_expected_response to all cases except one test_callTracer_type_create (we do not have tx_instruction for this case and I check response as it is)